spyder/utils/introspection/plugin_client.py

```diff
@@ -77,7 +81,7 @@ def run(self):
         # Set up environment variables.
         processEnvironment = QProcessEnvironment()
         env = self.process.systemEnvironment()
-        if (self.env and 'PYTHONPATH' not in self.env) or DEV:
+        if (self.env and 'PYTHONPATH' not in self.env) or self.env is None:
```

Comment:
hmm no, I tested and this condition evaluate to `True` when using `./bootstrap.py
`

I changed because previously it only evaluates to `True` when `DEV` was `True` (i. e. libraries, and spyder path only will be added in dev mode)